tests/Application/DonationTestDataTrait.php

```diff
@@ -93,4 +116,23 @@ protected function getAnonymousPendingTestDonation(): Donation
 
         return $donation;
     }
+
+    private function getCharity(): Charity
+    {
+        $charity = TestCase::someCharity();
+        $charity->setSalesforceId('123CharityId');
+        $charity->setName('Test charity');
+        $charity->setStripeAccountId('unitTest_stripeAccount_123');
+
+        return $charity;
```

Comment:
from the names it's not really clear to me what the difference in intent is between this `getCharity` and `TestCase::someCharity` - they both return charities with some arbitrary test data.

If it's just that the data needs to be different maybe clearer to decouple them enitely?

```suggestion
        return new Charity(
            salesforceId: ''123CharityId,
            charityName: "Test charity",
            stripeAccountId: 'unitTest_stripeAccount_123',
            hmrcReferenceNumber: 'H' . self::randomHex(3),
            giftAidOnboardingStatus: 'Onboarded',
            regulator: 'CCEW',
            regulatorNumber: 'Reg-no',
            time: new \DateTime('2023-10-06T18:51:27'),
        );
```

Also because eventually I'd like to lose any setter / mutator methods for things that we don't actually change in prod - like a charity's salesforceId never changes so I'd rather not have a setter for it. The less calls there are to that setter the easier it will be to remove.

Reply:
I don't think this needed any variation, just got muddled. Fixed on https://github.com/thebiggive/matchbot/pull/736/commits/8186b1591518f842ea771fbd58e1ec234a6e618e